### ISSUE TITLE
Trying the demo

### ISSUE THREAD

**Issue by yuzrieSiddiq** [benoitvallon/react-native-nw-react-calculator#35]
Hi, I managed to successfully run the demo using electron but not with react-native on Android. Here's what I get when trying with `react-native run-android`

```
╭─reis@Asakura ~/Documents/Testcode/react-native-nw-react-calculator  ‹master› 
╰─➤  react-native run-android                                            
Starting JS server...
Building and installing the app on the device (cd android && ./gradlew installDebug)...
:app:preBuild UP-TO-DATE
:app:preDebugBuild UP-TO-DATE
:app:checkDebugManifest
:app:preReleaseBuild UP-TO-DATE
:app:prepareComAndroidSupportAppcompatV72301Library UP-TO-DATE
:app:prepareComAndroidSupportRecyclerviewV72301Library UP-TO-DATE
:app:prepareComAndroidSupportSupportV42301Library UP-TO-DATE
:app:prepareComFacebookFrescoDrawee081Library UP-TO-DATE
:app:prepareComFacebookFrescoFbcore081Library UP-TO-DATE
:app:prepareComFacebookFrescoFresco081Library UP-TO-DATE
:app:prepareComFacebookFrescoImagepipeline081Library UP-TO-DATE
:app:prepareComFacebookFrescoImagepipelineOkhttp081Library UP-TO-DATE
:app:prepareComFacebookReactReactNative0210Library UP-TO-DATE
:app:prepareOrgWebkitAndroidJscR174650Library UP-TO-DATE
:app:prepareDebugDependencies
:app:compileDebugAidl UP-TO-DATE
:app:compileDebugRenderscript UP-TO-DATE
:app:generateDebugBuildConfig UP-TO-DATE
:app:generateDebugAssets UP-TO-DATE
:app:mergeDebugAssets UP-TO-DATE
:app:generateDebugResValues UP-TO-DATE
:app:generateDebugResources UP-TO-DATE
:app:mergeDebugResources UP-TO-DATE
:app:bundleDebugJsAndAssets SKIPPED
:app:processDebugManifest UP-TO-DATE
:app:processDebugResources UP-TO-DATE
:app:generateDebugSources UP-TO-DATE
:app:processDebugJavaRes UP-TO-DATE
:app:compileDebugJavaWithJavac UP-TO-DATE
:app:compileDebugNdk UP-TO-DATE
:app:compileDebugSources UP-TO-DATE
:app:preDexDebug UP-TO-DATE
:app:dexDebug UP-TO-DATE
:app:validateDebugSigning
:app:packageDebug UP-TO-DATE
:app:zipalignDebug UP-TO-DATE
:app:assembleDebug UP-TO-DATE
:app:installDebug
Installing APK 'app-debug.apk' on 'Nexus_5X_API_25_x86(AVD) - 7.1.1'
Installed on 1 device.

BUILD SUCCESSFUL

Total time: 9.438 secs

This build could be faster, please consider using the Gradle Daemon: http://gradle.org/docs/2.4/userguide/gradle_daemon.html
Starting the app (adb shell am start -n com.reactnativenwreactcalculator/.MainActivity)...
╭─reis@Asakura ~/Documents/Testcode/react-native-nw-react-calculator  ‹master› 
╰─➤ 
```

nothing shows on my emulator. I've turned it on via 
```
./emulator -avd Nexus_5X_API_25_x86
```

**Comment by ximet** [comment]
I think, this repository abandoned

**Comment by yuzrieSiddiq** [comment]
ah too bad then I'll just close it

**Comment by ximet** [comment]
@yuzrieSiddiq If you need help with examples react-native, I'll help you.

**Comment by yuzrieSiddiq** [comment]
alright thanks! I appreciate that very much. I've just started looking into react and react-native today so I'm really blur at a lot of stuffs. But for starters, I just want to make sure I can build the project. 

When I followed [React Native Docs](https://facebook.github.io/react-native/docs/getting-started.html), it just says I need to run `npm run android` but in the project's readme it shows `react-native run-android`. The problem here is I have no idea from where should I start looking at because there was no errors.
